l10n_eu_product_adr/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "ADR Dangerous Goods",
     "summary": "Allows to set appropriate danger class and components",
-    "version": "18.0.1.0.0",
+    "version": "18.0.1.0.1",
```

Comment:
there's no need to change the version of the module. The oca-bot will take care ;)